manifests/generated/vm-resource.yaml

```diff
@@ -127,6 +127,15 @@ spec:
                                 required:
                                 - name
                               type: array
+                            rng:
+                              properties:
+                                enabled:
+                                  type: boolean
+                                source:
```

Comment:
This is a security risk.

What would happen if you point this to /etc/my/secret/cert?